src/components/StatusBar/LogLevelSelect/index.tsx

```diff
@@ -169,6 +172,18 @@ const LogLevelSelect = () => {
         </Box>
     );
 
+    const updateFilter = useCallback((logLevels: LOG_LEVEL[]) => {
+        setSelectedLogLevels(logLevels);
+
+        // Filter logs in backend worker.
```

Comment:
```suggestion
```
This comment seems a bit redundant as the purpose can be inferred from the function's JsDoc. With that said, if the purpose is to differentiate from the React state setter, that's fine to keep.